components/UI/profileCard/profileCard.tsx

```diff
@@ -85,11 +85,11 @@ const ProfileCard: FunctionComponent<ProfileCard> = ({
           )}
         </div>
 
-          <div className="flex flex-col h-full justify-between">
+          <div className="flex flex-col h-full justify-center">
             <Typography type={TEXT_TYPE.BODY_SMALL} color="secondary" className={styles.accountCreationDate}>
               {sinceDate ? `${sinceDate}` : ""}
             </Typography>
-            <Typography type={TEXT_TYPE.H2} className={styles.profile_name}>{identity.domain.domain}</Typography>
+            <Typography type={TEXT_TYPE.H2} className={`${styles.profile_name} mt-2`}>{identity.domain.domain}</Typography>
```

Comment:
As outlined in [Tailwind CSS documentation](https://tailwindcss.com/docs/margin), the mt-2 class is a margin-top of 0.5rem